src/index.php

```diff
@@ -0,0 +1,305 @@
+                    });
+                    $scope.removeNotification = function (notification) {
+                        var index;
+                        if ($scope.notifications !== undefined) {
+                            index = $scope.notifications.indexOf(notification);
+                            $scope.notifications.splice(index, 1);
+                        }
+                    }
+                });
+
+                footerLinksApp.controller('MajikCtrl', function ($scope, $http, $rootScope) {
+                    var footerClass = null;
+                    var wp_ajax = function (_req) {
+                        _req.nonce = window.sb_nonce;
+                        return $http({
+                            method: 'POST',
+                            url: ajaxurl,
+                            data: jQuery.param(_req),
+                            headers: {'Content-Type': 'application/x-www-form-urlencoded'}
+                        })
+                    };
+                    $scope.patchTheme = function () {
+                        wp_ajax({
+                            action: 'NFL-patch_theme',
+                            data: $scope.footerClass
+                        }).success(function (res) {
+                            if (res.success) {
+                                $rootScope.$emit('notification', {
+                                    state: 'success',
+                                    msg: res.data
+                                });
+                            } else {
+                                $rootScope.$emit('notification', {
+                                    state: 'errror',
+                                    msg: res.data
+                                });
+                            }
+                        });
+                    };
+                });
+            })(angular, document, jQuery);
+        </script>
+        <h1>Remove Footer Links</h1>
+        <hr/>
+        <div ng-app="footerLinksApp">
+            <div ng-controller="MajikCtrl">
+                <h2>Input CSS class</h2>
+                <form name="theme_form" ng-submit="patchTheme()">
+                    <input type="text" name="nfl_theme" ng-model="footerClass" />
+                    <p class="submit"><input type="submit" class="button button-primary" value="Hide selected class"></p>
+                </form>
+            </div>
+            <div ng-controller="NotificationsCtrl">
+                <div class="notice notice-{{ n.state }}" ng-repeat="n in notifications">
+                    <p>{{n.msg}}
+                        <a ng-click="removeNotification(notification)">
+                            <span class="dashicons dashicons-no-alt"></span>
+                        </a>
+                    </p>
+                </div>
+            </div>
+        </div>
+        <?php
+    }
+}
+
+// Inside WordPress
+if (defined('ABSPATH')) {
+    $NiteowebFooterLinks_ins = new FooterLinks;
```

Comment:
Add namescape :) This is mostly because of this https://wordpress.org/plugins/search.php?type=term&q=Footer+Links (1000+ plugins that mig use that function name)